_inc/client/at-a-glance/akismet.jsx

```diff
@@ -33,6 +34,29 @@ class DashAkismet extends Component {
 		isDevMode: '',
 	};
 
+	constructor() {
+		super( ...arguments );
+
+		this.trackInstallClick = this.trackInstallClick.bind( this );
+		this.trackActivateClick = this.trackActivateClick.bind( this );
+	}
+
+	trackInstallClick() {
+		analytics.tracks.recordJetpackClick( {
+			type: 'install-link',
+			target: 'at-a-glance',
+			feature: 'akismet',
```

Comment:
I see that we were previously using `anti-spam` instead of `akismet` for the button that was removed in #12597. Does it matter?

Reply:
No issue re the button itself, but it is a great point to unify the props naming wrt to the `feature` prop as supplied to the `recordJetpackClick` f-n. Considering the prop refers to the overarching module, let's change it to `anti-spam` for consistency.

Reply:
We're updating this to be `anti-spam` in https://github.com/Automattic/jetpack/pull/12912